tests/test_parsing.py

```diff
@@ -8,7 +8,7 @@
 def read_test_data():
     """
     Expected files (tests/test_parsing):
-        mytest.py -- Python file calling valohai.prepare()
+        mytest.py -- Python (or .ipynb) file calling valohai.prepare()
```

Comment:
`Python (.py) or notebook (.ipynb) file calling valohai.prepare` would be more exact but I think most reading this will understand?